Wireframe/index.html

```diff
@@ -4,10 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+    <header>
+      <div>
+        <h1>Wireframe</h1>
+        <p>Discover the basics of Git, a powerful version control system used by developers to manage code, collaborate efficiently, and track changes across projects.</p>
+      </div>
+    </header>
+
+    <main>
+      <section>
```

Comment:
What is section doing in your code here? What is its function?

Reply:
The <section> tag groups related content together and provides a heading for that part. It helps organise the page better, making it easier for users and search engines to understand the content.

Reply:
The section tag groups related content together and provides a heading for that part. It helps organise the page better, making it easier for users and search engines to understand the content.

Reply:
Hi @SallyMcGrath, thanks for reviewing my code!

The section tag groups related content together and provides a heading for that part. It helps organise the page better, making it easier for users and search engines to understand the content.

Reply:
Here is the Lighthouse outcome:

Reply:
<img width="1203" alt="Screenshot 2024-10-16 at 12 30 06" src="https://github.com/user-attachments/assets/dd3fc7dd-dc2f-40e5-b25f-ee585476bbbc">